eo-phi-normalizer/src/Language/EO/Phi/Rules/Common.hs

```diff
@@ -238,9 +238,6 @@ equalBindings bindings1 bindings2 = and (zipWith equalBinding (sortOn attr bindi
 
 equalBinding :: Binding -> Binding -> Bool
 equalBinding (AlphaBinding attr1 obj1) (AlphaBinding attr2 obj2) = attr1 == attr2 && equalObject obj1 obj2
--- Ignore deltas for now while comparing since different normalization paths can lead to different vertex data
--- TODO #120:30m normalize the deltas instead of ignoring since this actually suppresses problems
-equalBinding (DeltaBinding _) (DeltaBinding _) = True
```

Comment:
Thanks! But without this line, is `equalBinding` any different from `(==)`?

Reply:
I believe so, because there is `equalObject` in the `AlphaBinding` case, which calls `equalBindings` which sorts the bindings. The `Eq` instance for `Binding` wouldn't do that and would call `(==)` for the objects as well

Reply:
Is this the only difference? @RoukayaZaki may I ask you to check this and add [Haddock documentation](https://haskell-haddock.readthedocs.io/) and [doctests](https://github.com/sol/doctest#readme) to all `equal*` functions in this module to make sure we do not break it in the future?

Reply:
Yeah I checked and I think it's the only difference. I will doctests soon in a separate PR to double check it.